### DISCUSSION TITLE
Bug on 404 page

### DISCUSSION THREAD

**Discussion by RuiAAPeres** [hugo-sid/hugo-blog-awesome#27] [General]
On small screens, the 404 page looks like this: 

![image](https://user-images.githubusercontent.com/949286/235689074-da4ae8a9-106e-43de-8c91-990d291bad2d.png)

Any suggestions on how to fix it? Thank you!

**Comment by hugo-sid** [comment]
Hi @RuiAAPeres, thanks for bringing this into notice.
I have pushed a commit to fix this. 

Happy blogging :)
Have a good day.
